### ISSUE TITLE
npm run init or flutter pub get versions issue

### ISSUE THREAD

**Issue by gumdum** [VNAPNIC/Flutter_Architecture#4]
When attempting to build the development environment I am getting conflicting version information. I've tried adjusting the different versions and simply removing the versions from pubspec.yaml file. 

I tried both npm run init, and flutter pub get.

This is the error trying to setup the development environment:
> init
> flutter pub get && npm run generate && npm run activateLocalization && npm run generateLocale

Because intl_translation >=0.17.7 depends on intl >=0.15.3 <0.17.0 and every version of flutter_localizations from sdk depends on intl 0.17.0, intl_translation >=0.17.7 is incompatible with flutter_localizations from sdk.
So, because flutter_architecture depends on both flutter_localizations any from sdk and intl_translation ^0.17.10+1, version solving failed.
Running "flutter pub get" in Flutter_Architecture...
pub get failed (1; So, because flutter_architecture depends on both flutter_localizations any from sdk and intl_translation ^0.17.10+1, version solving failed.)

**Comment by VNAPNIC** [comment]
first run project u can run "npm run setup" then "npm run init"
and u need to install the plugin https://plugins.jetbrains.com/plugin/13666-flutter-intl for the Android studio.

**Comment by VNAPNIC** [comment]
can U upgrade version intl_translation to 0.17.10+1 and intl_utils 2.1.0

**Comment by gumdum** [comment]
I installed the Flutter Intl from VS Code: https://marketplace.visualstudio.com/items?itemName=localizely.flutter-intl

I also added a lot of the dependencies to pubspec.yaml, and null'd out all versioning. Unfortunately the INTL packages are not stable and interchangeable at the moment.

I would recommend to make the 'Architecture' package as bare-bones as possible. Remove the fancy internationalization, and allow things like 'internationalization', 'database','session/token', etc to be added through an API framework.

**Comment by VNAPNIC** [comment]
u can config EVM flavor in here: [build_config.dart](https://github.com/VNAPNIC/Flutter_Architecture/blob/master/lib/config/src/build_config.dart)

intl_translation is good when u use it in the Android studio

**Comment by gumdum** [comment]
I was able to get it up and running, thank you for taking the time to assist.

It is running good on the following development environment:

OS: MacOS Big Sur 11.2.3
IDE: VS Code
Flutter version:  Stable 
-------
Flutter 2.0.3 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 4d7946a68d (4 days ago) • 2021-03-18 17:24:33 -0700
Engine • revision 3459eb2436
Tools • Dart 2.12.2
----------
I did change all the dependency versioning in pubspec.yaml to: null.  which removed some conflicting dependency errors.
![image](https://user-images.githubusercontent.com/71924534/112042066-6813d080-8b47-11eb-8c94-8a73d8774fb4.png)
Thanks and keep up the great work! I can't wait to start building with this foundation!
